kfs-core/src/main/resources/edu/arizona/kfs/sys/businessobject/datadictionary/ArizonaKfsKimAttributes.xml

```diff
@@ -0,0 +1,30 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
Please rename module KfsKimAttributes. We are trying to do away with the "AZ" or "Arizona" name prefaces.